### PR TITLE
Display user email for administrators

### LINKED ISSUE #1704: 🚀 [Feature] Make Email of a User visible for Moderators
## :rocket: Feature

A moderator needs to contact people. For now he can't. Therfore in the admin panels "Users" lookup we need this email address ether (and best) as a small icon which when be clicked it is copied to the clipboard if possible or with a hover.

### Design & Layout

The emails of users should **not** be directly listed, because of privacy.
user-lookup

### Additional context

As long as we do moderation by ourselfes this is ok. Later, if community moderation comes, we need ether an NDA for everybody or we need another method of communication

### PR DESCRIPTION
**Pull request by aonomike**
## 🍰 Pullrequest

- Display user email to administrators
- Refactor Permission middleware tests
- Add tests to check email of user only revealed to admin and owner

### Issues

fixes #1704 

### Images

Screen Shot 2019-10-05 at 5 29 31 PM

### CODE REVIEW

File: webapp/pages/admin/users.vue [hunk 1]
```diff
@@ -128,6 +138,7 @@ export default {
               id
               name
               slug
+              email
```
Comment (roschaefer) [inline webapp/pages/admin/users.vue]: At the moment, the backend does not allow anybody except the owner to see the `email` of the user. You will need to change permissions in the backend.

I think I would still feel safe, if only the owner *or* an admin user can see the email.

Comment (aonomike) [reply webapp/pages/admin/users.vue]: Done, thanks @roschaefer

**Comment by aonomike** [comment]
Screen Shot 2019-10-05 at 5 29 31 PM

File: webapp/pages/admin/users.vue [hunk 2]
```diff
@@ -33,6 +33,11 @@
             <b>{{ scope.row.name | truncate(20) }}</b>
           </nuxt-link>
         </template>
+        <template slot="email" slot-scope="scope">
+          <a :href="`mailto:${scope.row.email}`">
+            <b>{{ scope.row.email }}</b>
```
Comment (roschaefer) [inline webapp/pages/admin/users.vue]: Some of those emails are quite long. Maybe we should think about how to fit the data into the layout? E.g. at some places in the app, we use `| truncate(20)` filter.

Although I'm OK to merge this and wait for our admins to complain about it :wink:

Comment (mattwr18) [reply webapp/pages/admin/users.vue]: That's what I thought as well, @datenbrei mentioned that they might be copying and pasting the links, so maybe it's more difficult if it's truncated??

Comment (aonomike) [reply webapp/pages/admin/users.vue]: Thanks @mattwr18 @roschaefer 
Let me know if I need to update this :D

**Pull request merged**